kimchi/src/circuits/expr.rs

```diff
@@ -169,7 +169,7 @@ pub enum ConstantExpr<F> {
     EndoCoefficient,
     Mds { row: usize, col: usize },
     Literal(F),
-    Pow(Box<ConstantExpr<F>>, usize),
+    Pow(Box<ConstantExpr<F>>, u64),
```

Comment:
I don't think we ever use a large `Pow`, it might be safer to move to `u32`

Reply:
I kept `u64` as this is what arkworks expect when we use `pow` there, but I don't see a problem moving this to `u32` either